src/main/web/templates/auth-needed.ftl

```diff
@@ -0,0 +1,3 @@
+<html>
+You are authenticated3!
```

Comment:
This page is named `auth-needed.ftl` - is this the correct message then?

Reply:
Better name now.